### ISSUE TITLE
Layouting problem with multiple todos

### ISSUE THREAD

**Issue by Unaimend** [Otto-AA/dashy-todo#1]
Thanks for the package :). I found a small problem
```
== Mass spectroscopy
#todo[Descbribe what this is]
#todo[And how it works]
#todo[And how read the plots]
```
results in
![image](https://github.com/user-attachments/assets/8640557c-2ce5-42b0-9110-e19f80343701)

**Comment by Otto-AA** [comment]
Yeah, that looks not useful :upside_down_face: . The package currently does not compare the position of each todo against each other, it simply shows it on the side. I don't have the time and knowledge to implement such a placement algorithm currently.

If someone knows how to make it better, please give it a try!

For now, you likely can manually work around this issue, for instance:

```typst
#todo[
  - Describe what this is
  - And how it works
  - And how read the plots
]
```

or

```typst
== Mass spectroscopy
#todo[Describe what this is]
#v(2em)
#todo[And how it works]
#v(2em)
#todo[And how read the plots]
```
